src/main/kotlin/eda/shoppingBasket/service/entity/Offering.kt

```diff
@@ -0,0 +1,18 @@
+package eda.shoppingBasket.service.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.ManyToOne
+import java.util.UUID
+
+@Entity
+class Offering(
+    val name: String,
+    val unitPrice: Float,
+    val totalPrice: Float,
+    @Id
+    val offeringID: UUID,
+    @ManyToOne
+    val product: Product
```

Comment:
We can leave out product I think (it's only the name?)

Reply:
yep. removed it